src/Symfony/Bundle/WebServerBundle/WebServerBundle.php

```diff
@@ -11,8 +11,13 @@
 
 namespace Symfony\Bundle\WebServerBundle;
 
+use Symfony\Component\Console\Application;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 class WebServerBundle extends Bundle
 {
+    public function registerCommands(Application $application)
+    {
+        // prevents convention based registration of console commands as they are registered as services
```

Comment:
well, the existing logic already prevents it if they are defined as services

Reply:
indeed, useless. reverted